.azure-pipelines/scripts/sqlserver/windows/32_install_odbc.bat

```diff
@@ -1,5 +1,5 @@
 :: https://docs.microsoft.com/en-us/sql/connect/odbc/windows/system-requirements-installation-and-driver-files
 :: Finding the actual URL not gated by a form was a nightmare
```

Comment:
haha

Reply:
It was much easier once Ofek had done all the hard work :)